src/components/Tool/Appeal/Modals/DeleteAppealContact/DeleteAppealContactModal.test.tsx

```diff
@@ -218,4 +223,38 @@ describe('DeleteAppealContactModal', () => {
     });
     expect(handleClose).toHaveBeenCalledTimes(1);
   });
+
+  it('should not refetch the flows columns after mutation', async () => {
```

Comment:
Can we add a little more to this title to distinguish it from the next test? Maybe `should not refetch the flows columns after mutation when not in flows view`